certified-connectors/LMS365/apiDefinition.swagger.json

```diff
@@ -3786,8 +4096,7 @@
         ],
         "responses": {
           "default": {
```

Comment:
@elearningforce-ag Can you please update this last one "default" response with expected status code as you did for other "default" responses.

Reply:
This is generic "HttpRequest" method and can be used to perform any request to our API. So there is no way to define any expected status codes here.